sdk/contentunderstanding/Azure.AI.ContentUnderstanding/src/Generated/ContentUnderstandingClient.cs

```diff
@@ -0,0 +1,1296 @@
+        /// <exception cref="ArgumentNullException"> <paramref name="analyzerId"/> or <paramref name="resource"/> is null. </exception>
+        /// <exception cref="ArgumentException"> <paramref name="analyzerId"/> is an empty string, and was expected to be non-empty. </exception>
+        public virtual Operation<ContentAnalyzer> CreateAnalyzer(WaitUntil waitUntil, string analyzerId, ContentAnalyzer resource, bool? allowReplace = default, CancellationToken cancellationToken = default)
+        {
+            Argument.AssertNotNullOrEmpty(analyzerId, nameof(analyzerId));
+            Argument.AssertNotNull(resource, nameof(resource));
+
+            Operation<BinaryData> result = CreateAnalyzer(waitUntil, analyzerId, resource, allowReplace, cancellationToken.ToRequestContext());
+            return ProtocolOperationHelpers.Convert(result, response => ContentAnalyzer.FromLroResponse(response), ClientDiagnostics, "ContentUnderstandingClient.CreateAnalyzer");
+        }
+
+        /// <summary> Create a new analyzer asynchronously. </summary>
+        /// <param name="waitUntil"> <see cref="WaitUntil.Completed"/> if the method should wait to return until the long-running operation has completed on the service; <see cref="WaitUntil.Started"/> if it should return after starting the operation. For more information on long-running operations, please see <see href="https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/LongRunningOperations.md"> Azure.Core Long-Running Operation samples</see>. </param>
+        /// <param name="analyzerId"> The unique identifier of the analyzer. </param>
+        /// <param name="resource"> The resource instance. </param>
+        /// <param name="allowReplace"> Allow the operation to replace an existing resource. </param>
+        /// <param name="cancellationToken"> The cancellation token that can be used to cancel the operation. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="analyzerId"/> or <paramref name="resource"/> is null. </exception>
+        /// <exception cref="ArgumentException"> <paramref name="analyzerId"/> is an empty string, and was expected to be non-empty. </exception>
+        public virtual async Task<Operation<ContentAnalyzer>> CreateAnalyzerAsync(WaitUntil waitUntil, string analyzerId, ContentAnalyzer resource, bool? allowReplace = default, CancellationToken cancellationToken = default)
+        {
+            Argument.AssertNotNullOrEmpty(analyzerId, nameof(analyzerId));
+            Argument.AssertNotNull(resource, nameof(resource));
+
+            Operation<BinaryData> result = await CreateAnalyzerAsync(waitUntil, analyzerId, resource, allowReplace, cancellationToken.ToRequestContext()).ConfigureAwait(false);
+            return ProtocolOperationHelpers.Convert(result, response => ContentAnalyzer.FromLroResponse(response), ClientDiagnostics, "ContentUnderstandingClient.CreateAnalyzerAsync");
```

Comment:
Emitter bug.  LRO resource creation with [PUT](https://github.com/microsoft/api-guidelines/blob/vNext/azure/ConsiderationsForServiceDesign.md#create-or-replace-a-resource-requiring-additional-long-running-processing) returns the resource JSON without status monitor wrapper.  FromLroResponse() should not expect `result`.